scripts/test/run_selenium_tests_dev.sh

```diff
@@ -21,4 +21,5 @@ docker-compose run \
    -e ELASTICSEARCH_INDEX=testindex \
    -e USE_WEBPACK_DEV_SERVER=True \
    -e WEBPACK_DEV_SERVER_HOST="$WEBPACK_SELENIUM_DEV_SERVER_HOST" \
+   -e ELASTICSEARCH_DEFAULT_PAGE_SIZE=5 \
```

Comment:
i think it's probably time to stick these env values in a separate settings file rather than requiring a shell script to specify them in a series of `-e`'s